docs/requirements.txt

```diff
@@ -1,19 +1,29 @@
-celery
 boto3
+celery
 dogpile.cache
 flask
 flask-login
 flask-migrate
 flask-sqlalchemy
+opentelemetry-api
+opentelemetry-exporter-otlp
+opentelemetry-instrumentation
+opentelemetry-instrumentation-botocore
+opentelemetry-instrumentation-celery
+opentelemetry-instrumentation-flask
+opentelemetry-instrumentation-logging
+opentelemetry-instrumentation-requests
+opentelemetry-instrumentation-sqlalchemy
+opentelemetry-instrumentation-wsgi
+opentelemetry-sdk
 operator-manifest
-python-qpid-proton
-python-memcached
 psycopg2-binary
+python-memcached
+python-qpid-proton
 recommonmark
 requests
 ruamel.yaml
-# https://github.com/sphinx-doc/sphinx/issues/2796
-sphinx==6.2.0
+sphinx==6.2.1
```

Comment:
May I ask you a favor?
Could you test latest sphinx release - v7.0.1 I would go with pinning latest rather then older one. 
But only if everything passes.

Reply:
_# And also dependabot is suggesting to pin latest version in different PR._

Reply:
_# And also dependabot is suggesting to pin latest version in different PR._

Reply:
I already wrote the reason why this is not possible in the PR description ([sphinx-rtd-theme issue](https://github.com/readthedocs/sphinx_rtd_theme/issues/1463)).

Reply:
I should do the review better next time - starting reading description :D ... thanks!